vsts_ci/win32/continuous-build-win32.yml

```diff
@@ -0,0 +1,16 @@
+steps:
+  - script: |
+      choco install python2
```

Comment:
Can we use the `UsePythonVersion` task instead of handling Python installation ourselves?

Reply:
no. there is no cache for python in win1803

Reply:
Got it.